platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -515,19 +513,6 @@ public SwirldsPlatform(@NonNull final PlatformComponentBuilder builder) {
             eventStreamManagerName = String.valueOf(selfId);
         }
 
-        final EventStreamManager eventStreamManager = new DefaultEventStreamManager(
-                platformContext,
-                time,
-                threadManager,
-                getSelfId(),
-                this,
```

Comment:
Optional: make `SwirldsPlatform::sign` throw an `UnsupportedOperationException`, since I think it shouldn't ever be used

Reply:
It turns out this is actually still used by the record stream. We can refactor this dependency away, but it probably belongs in a stand alone PR.